### ISSUE TITLE
Mara Chain Documentation - Developers Guide

### ISSUE THREAD

**Issue by Olanetsoft** [mara-labs/mara-chain#54]
__Proposal:__ 

I am proposing a separate GitHub repository for our developer guide documentation. This will help to keep the documentation organized and easily accessible for all of our contributors and users.

I propose we use Docusaurus to power our documentation and host it on a domain such as docs.mara.xyz. **Docusaurus** is a popular open-source static site generator that is easy to use and customize, making it an excellent choice for our documentation. Hosting the documentation on a separate domain will make it more accessible and user-friendly.

By creating a separate repository for our documentation, we can ensure that it stays up-to-date and accurate, making it easier for our developers to find the information they need. Additionally, using Docusaurus will make it easy for us to update the documentation and add new pages.

I would love to hear your thoughts and feedback on this proposal. If you have any questions or concerns, please don't hesitate to let me know.

Let's work together to make our documentation the best it can be!

__Current behavior:__ 

Current documentation powered by Vuepress should remain on the Chain docs to guide anyone with a technical background or a validator. The only update should be the design guide, template, theme etc.

**Comment by blewater** [comment]
Makes sense to me to have a separate Web3 documentation repo for smart contract developers targeting the Chain EVM. A clarification point, there's a Mara-chain repo for the validators and node operators [mara-labs/networks](https://github.com/mara-labs/networks) containing genesis, network information, and Validating documentation.

**Comment by Olanetsoft** [comment]
> Makes sense to me to have a separate Web3 documentation repo for smart contract developers targeting the Chain EVM. A clarification point, there's a Mara-chain repo for the validators and node operators [mara-labs/networks](https://github.com/mara-labs/networks) containing genesis, network information, and Validating documentation.

Good.
